modules/web/src/app/kubeone-wizard/template.html

```diff
@@ -0,0 +1,109 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<form [formGroup]="form">
+  <mat-card fxFlex="100%">
+    <div fxLayout="row">
+      <mat-card-header class="km-no-padding"
+                       fxLayout="row"
+                       fxLayoutGap="30px">
+        <mat-card-title>
+          <div>Import KubeOne Cluster</div>
```

Comment:
I don't think there is a design for `Create` (at least I didn't find one). For import we have this design: https://www.figma.com/file/dJQKgUPzIDND36o2Nq8dct/Loodse-UI?node-id=6516%3A54056

And this will be `Import` because we're importing the KubeOne clusters and not creating them.